triaging.rst

```diff
@@ -43,11 +43,32 @@ or a veteran core developer, they're actively choosing to voluntarily donate the
 time towards the improvement of Python. As is the case with any member of
 the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
-.. note::
+Becoming a member of Python Triage team
+---------------------------------------
+
+Any Python core developers are welcome to invite a Python contributor to the
+Python Triage team. Do note that the responsibilities of a Python Triage member
+is more elevated than a bug triager in bpo. For example, the Python Triage member
+has access to more repositories than just CPython. Triages will be responsible
+to handle not just issues, but also pull requests, and even commanding backports.
+
+Any existing triagers in b.p.o can transition to the Python Triage role.
```

Comment:
Even simpler: 
```suggestion
Any existing triager in b.p.o can become a Python triager.
```

Reply:
Personally, I think that we should refer a member of the role on on bpo as "developer". That is the official name of the role on there, and it also helps to avoid confusing it with the new Python triagers on GitHub.